apps/condo/domains/news/components/NewsForm/NewsItemSharingForm.tsx

```diff
@@ -13,6 +13,9 @@ import { IFrame } from '@condo/domains/miniapp/components/IFrame'
 import { MemoizedSharingNewsPreview } from '@condo/domains/news/components/NewsPreview'
 import { NEWS_TYPE_EMERGENCY } from '@condo/domains/news/constants/newsTypes'
 
+import { NewsItemSharingCustomRecipientCounter } from '../RecipientCounter'
+import { TNewsItemScopeNoInstance } from '../types'
```

Comment:
For names of types and interfaces we usually use prefix `I` or postfix `Type`. 
I have not yet encountered prefix `T`

Reply:
Agreed that this syntax is a bit wierd :-) Sometimes it is used in condo: TTextHighlighterRenderPartFN

Fixed it